modules/cnv-cdi-scratch-space-support-matrix.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-cdi-scratch-space.adoc
+
+[id="cnv-cdi-supported-operations-matrix_{context}"]
+= CDI supported operations matrix
+
+This matrix shows the supported CDI operations for differeent content types, and which of these operations requires scratch space.
+
+|===
+|Content types | HTTP | HTTPS | HTTP basic auth | Registry | S3 Bucket | Upload
+
+| KubeVirt(QCOW2)        
+|&#10003; QCOW2 +
```

Comment:
when viewing this file, the check mark (✓) is in a separate line (and not next to "qcow2")

Reply:
How are you viewing this file?
This renders as expected in [github](https://github.com/openshift/openshift-docs/blob/f997fe103260abba0b42f5b55ebf00592a0f2be2/modules/cnv-cdi-scratch-space-support-matrix.adoc) and in a local build - you can view a copy [here](http://file.bne.redhat.com/aburden/CNV-docs-preview/cnv1523-scratchspace/cnv1523-scratchspace/cnv/cnv_users_guide/cnv-cdi-supported-operations.html#cnv-cdi-supported-operations-matrix_cnv-cdi-supported-operations)